### ISSUE TITLE
Lack of Focus on Birth Certificates.mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#149]
Screengrab/videos (insert file names here)
2020 08 25 Lack of Focus on Birth Certificates.mp4
[2020 08 25, Lack of Focus on Birth Certificate.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5125132/2020.08.25.Lack.of.Focus.on.Birth.Certificate.mp4.zip)

Link
https://registry.boston.gov/birth

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.5 > Chrome 84

Trusted Tester Test
n/a

Brief Description
While testing the accessibility of the site, lost Focus as the cursor cannot be seen.

Steps to reproduce
1)	00:00, Log onto site, using the link provided above
2)	00:14, Starting at City of Boston link to tab through site
3)	00:20, Tabbed to the “Mine” button on the Radio button set. 
4)	00:24, To move to the “Someone Else” use the up/down arrow keys.
5)	00:27, Tab to the “Spouse” section
6)	00:29, To get to “child” use the up/down arrow keys
7)	00:34, Press the tab key to move to “Next Question”
Note This is a bug as the Focus is not easily visible.  Also note that Metrolist used a Red Border to indicate Focus.

**Comment by subaha-cob** [comment]
@andrebtlima moving back to "Needs triage". Will work on it when I work on registry app tickets
